### ISSUE TITLE
Wiki is outdated

### ISSUE THREAD

**Issue by aubreyyan** [UniversalDataTool/universal-data-tool#536]
**The bug:**
The github wiki provides an outdated implementation of `UniversalDataTool`, which has since been moved to `UniversalSampleEditor`, rendering these docs inaccurate and confusing for new users

**Steps to Reproduce?**
[Minimal example of code that reproduces the behavior](https://github.com/UniversalDataTool/universal-data-tool/wiki/Usage-with-React)

**Comment by illtellyoulater** [comment]
Hello @aubreyyan, new user here and I'm already completely confused.

The README says that in order to run the web-app locally I should just do `npm install` and then `npm run start`, however when I do so I receive the following errors messages:

```
PS C:\Users\my-username\downloads\universal-data-tool-0.14.26> npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: universal-data-tool@0.14.26
npm ERR! Found: webpack@5.72.1
npm ERR! node_modules/webpack
npm ERR!   peer webpack@">=2" from babel-loader@8.2.5
npm ERR!   node_modules/babel-loader
npm ERR!     peerOptional babel-loader@"^=8.x" from @cypress/react@4.16.4
npm ERR!     node_modules/@cypress/react
npm ERR!       dev @cypress/react@"^4.16.4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional webpack@"^=3.x" from @cypress/react@4.16.4
npm ERR! node_modules/@cypress/react
npm ERR!   dev @cypress/react@"^4.16.4" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\my-username\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\my-username\AppData\Local\npm-cache\_logs\2022-05-27T01_31_09_817Z-debug-0.log
```

I'm on Windows 10, running node v16.15.0.
I've rarely used node.js so I have no idea about the severity of the those issues... if they indicate some serious issue with the app code, or if I can just ignore them... And yes, as you said the documentation it's a bit of a mess, instructions from a part of the website are contradicting other instructions here at the repository... It's very frustrating...
Also I was expecting to find a Windows Desktop Client in the Releases section... At least that's what the website says... but it's  just the web-app and nothing else...

Hopefully you can share a couple of pointers...?
Thanks a lot if you can help!

**Comment by lmmx** [comment]
Yeah I get the same error with 0.14.25, 0.14.24, etc. I ended up using the 0.14.16 release (which has AppImage and other binaries) https://github.com/UniversalDataTool/universal-data-tool/releases/tag/v0.14.16

In this thread https://github.com/UniversalDataTool/universal-data-tool/issues/338 it's mentioned that it may be due to "a problem with the build server"

**Comment by illtellyoulater** [comment]
@lmmx glad you got it to work... I'll try again following your example when I'll get back to this... but I am afraid that in the meantime other solution will come available. If you get to know about any other good alternatives and you're around these lands perhaps give me a shout! ;)

**Comment by lmmx** [comment]
> @lmmx glad you got it to work... I'll try again following your example when I'll get back to this... but I am afraid that in the meantime other solution will come available. If you get to know about any other good alternatives and you're around these lands perhaps give me a shout! ;)

I'm using Label Studio now

**Comment by illtellyoulater** [comment]
@lmmx perfect, thanks for the info!
